android/samples/listbookmarkscreenlet/src/main/java/com/liferay/mobile/screens/listbookmark/BookmarkListInteractor.java

```diff
@@ -17,12 +17,14 @@
 	protected JSONArray getPageRowsRequest(Query query, Object... args) throws Exception {
 		long folderId = (long) args[0];
 
+		if (args[1] != null) {
+			query.setComparator((String) args[1]);
```

Comment:
What happen when we have a Liferay 6.2 instance and we set a Liferay 7 comparator?